homeassistant/components/panasonic_viera/config_flow.py

```diff
@@ -31,19 +32,21 @@ class PanasonicVieraConfigFlow(ConfigFlow, domain=DOMAIN):
 
     VERSION = 1
 
+    _remote: RemoteControl | None = None
+
     def __init__(self) -> None:
         """Initialize the Panasonic Viera config flow."""
-        self._data = {
+        self._data: dict[str, Any] = {
             CONF_HOST: None,
             CONF_NAME: None,
             CONF_PORT: None,
             CONF_ON_ACTION: None,
             ATTR_DEVICE_INFO: None,
         }
 
-        self._remote = None
-
-    async def async_step_user(self, user_input=None):
+    async def async_step_user(
+        self, user_input: dict[str, Any] | None = None
+    ) -> ConfigFlowResult:
         """Handle the initial step."""
         errors = {}
```

Comment:
```suggestion
        errors: dict[str, str] = {}
```
_Just a personal preference. I like to add `dict[str, str] for `errors`. It's not fully necessary._